### ISSUE TITLE
[Keyboard Navigation - Social Embedded-Edit Profile]: There is no consistent navigation on "Edit Profile" page.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#14]
"Priority: Low

Repro Steps
1. Install ""One Bus Away"" app on Android.  
2. Start ""One Bus Away"" app from Android phone.
3. ""One Bus Away "" Home page will get appear.
4. Navigate to ""Open Navigation Drawer"" button and press ENTER (or double tap) to open.
5. Navigate to ""Sign In"" and press ENTER (or double tap) to open. 
6. Navigate to ""Sign In with google"" and press ENTER (or double tap) to open.
7. Login with valid google account credentials.
8. “Create Profile” page will get appear.
9. Fill all the input fields with valid inputs.
10. Navigate to ""Done"" button and press ENTER.
11. Navigate to ""Open Navigation Drawer"" button and press ENTER  (or double tap).
12. Navigate to ""My Profile"" and press ENTER  (or double tap) to open.
13. Navigate to  ""Edit Profile"" and press ENTER  (or double tap) to open.
14. Navigate the page via ""TAB"" key .
15. Verify the consistent navigation on page.

Expected
While navigating via TAB key, when focus goes on edit box if device keyboard is display then it should be display for all edit box, if not then it should not display for all edit box. It means there should be consistency in navigation.

Actual
While navigating via TAB key, when focus goes on edit box then device keyboard is displaying for ""Last Name"" & ""Bio(Optional)"" edit box, while keyboard is not displaying on ""First Name"" edit box.

MAS Rule: MAS3.2.3

Impact
User would get confuse and disorient, If the device keyboard is opens on ""First Name"" edit box, but not for other edit box.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : Talk Back"
